src/invidious/playlists.cr

```diff
@@ -379,12 +374,15 @@ def fetch_playlist(plid, locale)
     author_thumbnail = ""
     ucid = ""
   else
-    author_info = playlist_sidebar_renderer[1]["playlistSidebarSecondaryInfoRenderer"]?.try &.["videoOwner"]["videoOwnerRenderer"]?
+    author_info = playlist_sidebar_renderer[1].dig?(
```

Comment:
`[1]` could be added to the dig

Reply:
given that the `if` right above ensurtes that we have `playlist_sidebar_renderer.size >= 2`, I though that for style, it could be left as is (the `1` alone at the beginning of the line was looking sad ^^>)